gooddata-java/src/test/java/com/gooddata/sdk/service/gdc/DatastoreServiceAT.java

```diff
@@ -12,16 +12,12 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
avoid star import at least in this case - `java.io.*` actually can be ton of things

Reply:
fixed